app/models/rummager_search.rb

```diff
@@ -32,6 +41,20 @@ def start
     search_result["start"]
   end
 
+  def organisations
+    @_organisations ||= search_result.dig('aggregates', 'organisations', 'options').map do |option|
+      organisation = option['value']
+      Organisation.new(
```

Comment:
Given the attribute names of the `Organisation` class are identical to the keys here you could probably abbreviate this to:
```ruby
attr_keys = %w(title content_id link slug organisation_state documents)
Organisation.new(organisation.slice(*attr_keys))
```
or similar.

Reply:
The keys are slightly different than the attr name. I could change the attr name 'document_count' to 'documents'. 
But the organisation variable does not have the document key value pair on it so the above code does not quite work

Reply:
```ruby
keys = %w(title content_id link slug organisation_state)
attrs = organisation.slice(*keys).merge('document_count' => option['documents'])
Organisation.new(attrs)
```
or similar, though perhaps this is getting overly complex, you might be better off just being explicit about it.

Reply:
I was going to do the above then I thought it was getting over complicated to read, so I am going to leave it as is.